tests/platform_tests/api/test_sfp.py

```diff
@@ -718,9 +718,10 @@ def test_reset(self, request, duthosts, enum_rand_one_per_hwsku_hostname, localh
                 continue
             info_dict = port_index_to_info_dict[sfp_port_idx]
 
-            # only flap interfaces where are CMIS optics,
-            # non-CMIS optics should stay up after sfp_reset(), no need to flap.
-            if "cmis_rev" in info_dict:
+            # CMIS optics need to be flapped after sfp_reset to come up.
```

Comment:
@arista-nwolfe what if the optics is CMIS but DAC, should we not skip the test?
Also we should check if CMIS compliant optics support low power mode?

Reply:
Yeah I think this is a fair point.
In the case of a CMIS but DAC we don't need to flap the port.
So I think we can improve this condition to only check for `is_xcvr_support_lpmode`.
I'll update the review.